crates/wasm-smith/src/config.rs

```diff
@@ -82,6 +133,18 @@ pub trait Config: Arbitrary + Default {
         1
     }
 
+    /// The maximum, in 64k Wasm pages, of any memory's initial or maximum size.
+    /// Defaults to 2^16 = 65536 (the maximum possible for 32-bit Wasm).
+    fn max_memory_pages(&self) -> u32 {
```

Comment:
Can you hook this up to `SwarmConfig` too?

Reply:
Done!